clients/drcachesim/reader/reader.h

```diff
@@ -102,6 +102,14 @@ class reader_t : public std::iterator<std::input_iterator_tag, memref_t>,
     virtual reader_t &
     operator++();
 
+    // Skips records until "count" instruction records have been passed.
+    // This will skip top-level headers for a thread; it is up to the caller
+    // to first observe those before skipping, if needed.  For interleaved-thread
+    // iteration, top-level headers in other threads will be skipped as well
+    // (but generally speaking these are identical to the initial thread).
```

Comment:
Also add that the caller can use the memtrace_stream_t interface to get them later.

Reply:
Done, w/ a TODO since that will be done separately.